CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### Changed
+
+- Starting from `v1.21.0` of semantic conventions,
+  `go.opentelemetry.io/otel/semconv/{version}/httpconv`
+  and `go.opentelemetry.io/otel/semconv/{version}/netconv`
+  packages will no longer be published. (#4145)
```

Comment:
In theory this is not needed (as this is for future package). However,  I think "any message" is better than "no message".